packages/flutter_tools/test/general.shard/plugins_test.dart

```diff
@@ -47,6 +48,8 @@ class _PluginPlatformInfo {
   /// The package entry for an Android plugin implementation using pluginClass.
   final String? androidPackage;
 
+  final bool? sharedDarwinSource;
```

Comment:
Same here; can this be a default-false non-nullable?

Reply:
Done.